mitmproxy/addons/tlsconfig.py

```diff
@@ -583,23 +586,46 @@ def _warn_seclevel_missing(self, side: Literal["client", "server"]) -> None:
                 f"for insecure TLS versions to work."
             )
 
+    @property
+    def crl_path(self) -> str:
+        if not self.certstore:
+            return ""
+
+        return "/" + str(self.certstore.default_ca.serial) + ".crl"
```

Comment:
Would `"/mitmproxy-" + str(self.certstore.default_ca.serial) + ".crl"` make sense to make it clear what's happening here to someone who inspects the cert?